### ISSUE TITLE
Let people sign up with their Google account?

### ISSUE THREAD

**Issue by Papaya-Voldemort** [skbidisigma1/stocks-sim#4]
I don't know how hard this would be, but could you make it so I could sign in with my google account?
Also you do know you can edit GitHub on Chrombook?

**Comment by skbidisigma1** [comment]
Yeah that will probably make this easier, I'll show you what we have so far later today but we can get banana on this soon

**Comment by skbidisigma1** [comment]
And by edit on Chromebook you mean work on the repo on Chromebook? That's usually how I do it

**Comment by skbidisigma1** [comment]
I fixed the security vulnerabilities we should be all good now
